apps/full-stack-tests/src/unified-selection/SelectionScope.test.ts

```diff
@@ -33,7 +33,6 @@ describe("SelectionScope", () => {
         cacheDir: path.join(__dirname, ".cache", `${process.pid}`),
       },
     });
-    // eslint-disable-next-line @itwin/no-internal
     RpcManager.registerImpl(ECSchemaRpcInterface, ECSchemaRpcImpl);
```

Comment:
I'm surprised ESLint says the rule is not necessary - both `ECSchemaRpcInterface` and `ECSchemaRpcImpl` are internal. @wgoehrig, have you noticed anything like this in the core repo? Any ideas what made the rule think this is not usage of internal API?

Reply:
Looks like `no-internal` rule only catches internal function/method invocation or internal class object creation. Filled an issue in eslint-plugin repo: https://github.com/iTwin/eslint-plugin/issues/84